src/components/calcite-tab-title/calcite-tab-title.scss

```diff
@@ -1,124 +1,113 @@
-$tab-margin: 1.25rem;
-
 :host {
-  flex: 0 1 auto;
-  outline: none;
-  margin-right: $tab-margin;
-  margin-inline-start: 0;
-  margin-inline-end: $tab-margin;
+  @apply flex-initial outline-none mx-5;
+  margin-inline-start: theme("margin.0");
+  margin-inline-end: theme("margin.5");
 }
 
 :host([layout="center"]) {
+  @apply text-center my-0 mx-5;
   flex-basis: 200px;
-  text-align: center;
-  margin: 0 $tab-margin;
 }
 
 :host([position="below"]) a {
-  border-bottom: 0;
-  border-top: 3px solid transparent;
+  @apply border-b-0
+    border-t-2
+    border-t-color-transparent;
+  border-top-style: solid;
 }
 
 :host a {
-  @include focus-style-base();
+  @apply focus-base;
 }
 
 :host(:focus) a {
-  @include focus-style-outset();
+  @apply focus-outset;
 }
 
 :host(:active),
 :host(:focus),
 :host(:hover) {
   a {
-    text-decoration: none;
-    color: var(--calcite-ui-text-1);
-    border-color: var(--calcite-ui-border-2);
+    @apply no-underline text-color-1 border-color-2;
   }
 }
 
 :host([active]) a {
-  color: var(--calcite-ui-text-1);
-  border-color: transparent;
+  @apply text-color-1 border-color-transparent;
 }
 
 :host([disabled]) {
-  pointer-events: none;
+  @apply pointer-events-none;
   span,
   a {
-    pointer-events: none;
-    opacity: var(--calcite-ui-opacity-disabled);
+    @apply pointer-events-none opacity-50;
   }
 }
 
 a,
 span {
-  box-sizing: border-box;
-  @include font-size(-2);
-  padding: $baseline/2 0;
-  white-space: nowrap;
-  overflow: hidden;
-  text-overflow: ellipsis;
-  border-bottom: 3px solid transparent;
-  -webkit-appearance: none;
-  cursor: pointer;
-  transition: $transition;
-  color: var(--calcite-ui-text-3);
-  width: 100%;
-  height: 100%;
-  display: flex;
-  justify-content: center;
+  @apply box-border
+    appearance-none
+    cursor-pointer
+    truncate
+    border-b-2
+    border-b-color-transparent
+    w-full
+    h-full
+    flex
+    justify-center
+    text-color-3
+    text--1h
+    transition-default
+    px-0
+    py-2;
```

Comment:
p-0

Reply:
But padding wasn't applied to all sides of the link tag previously, and wasn't 0:
```scss
  padding: $baseline/2 0; // ie., `0.75rem 0;`
```

Initially, I used
```scss
 px-0
 py-3 // 0.75rem
```
But @bstifle told me to use spacing 2 from the theme (0.5rem), thus I went with `py-2` (0.5rem).
I can change this back to `py-3` though - if we prefer more spacing around the tab title.

Reply:
Oh, fail on my part :facepalm:  i read that as px-0 py-0